yjit/src/options.rs

```diff
@@ -30,8 +30,9 @@ pub struct Options {
     /// Dump compiled and executed instructions for debugging
     pub dump_insns: bool,
 
-    /// Dump all compiled instructions in inlined CodeBlock
-    pub dump_disasm: bool,
+    /// Dump all compiled instructions. Some(false) dumps only inlined cb,
+    /// and Some(true) dumps both inlined and outlined cbs.
+    pub dump_disasm: Option<bool>,
```

Comment:
Could you use either an enum or a string for `dump_disasm`. `Option<bool>` is kind of cryptic IMO.

Reply:
sure, like this? https://github.com/ruby/ruby/pull/6309/commits/bccd1519e06919a55e61a1b923766bd547d83ba8